index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header class="Top"><h1>My Portfolio</h1></header>
+        <nav class="menu">
+            <img class="logo" src="images/toolbar-mobile.png" alt=""><a href="#"></a>       
+        </nav>
+        <div class="intro">
+            <h1>Hi, My name is Jorum and I am glad to see you!</h1>
+        </div>
+        <div class="details">
+            <p>I am a software developer! I can help you build a product.,
+                feature or website. look through some of my work and experience!
+                If you like what you see and you have a project you need coded,
+                do not hesitate to contact me.
+            </p>
+        </div>
+        <div class="connect">
+            <p>LET'S CONNECT</p>
+        </div>
+        <div class="connect-logo">
+            <img src="images/twitterIcon.svg" alt=""><a href="#"></a>
+            <img src="images/linkedInIcon.png" alt=""><a href="#"></a>
+            <img src="images/mIcon.svg" alt=""><a href="#"></a>
+            <img src="images/gitHubIcon.svg" alt=""><a href="#"></a>
+            <img src="images/peaceIcon.svg" alt=""><a href="#"></a>
+        </div>
```

Comment:
- [ ] You should use `<ul>` and `<li>` tags for this list because naturally this is unordered list and the semantically appropriate tag to use here is `<ul>` tag.
- [ ] You should also wrap the images within an `<a>` tag so that any visitor who clicks them would be navigated to your social media profiles. For example:
```
<ul>
  <li> <a> <img src=""></a></li>
  <li>...</li>
</ul>
```